tests/constantinople/eip1014_create2/test_create_returndata.py

```diff
@@ -0,0 +1,106 @@
+"""
+Return data management around create2
+Port call_outsize_then_create2_successful_then_returndatasizeFiller.json test
+Port call_then_create2_successful_then_returndatasizeFiller.json test
+"""
+
+import pytest
+from ethereum.crypto.hash import keccak256
+
+from ethereum_test_tools import Account, Alloc, Environment, StateTestFiller, Transaction
+from ethereum_test_tools.vm.opcode import Bytecode
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+from .spec import ref_spec_1014
+
+REFERENCE_SPEC_GIT_PATH = ref_spec_1014.git_path
+REFERENCE_SPEC_VERSION = ref_spec_1014.version
+
+
+@pytest.mark.valid_from("Istanbul")
+@pytest.mark.parametrize("call_return_size", [35, 32, 0])
+@pytest.mark.parametrize("create_type", [Op.CREATE, Op.CREATE2])
+def test_create2_return_data(
+    call_return_size: int,
+    create_type: Op,
+    pre: Alloc,
+    state_test: StateTestFiller,
+):
+    """
+    Validate that create2 return data does not interfere with previously existing memory
+    """
+    # Storage vars
+    slot_returndatasize_before_create = 0
+    slot_returndatasize_after_create = 1
+    slot_return_data_hash_before_create = 2
+    slot_return_data_hash_after_create = 3
+    slot_code_worked = 4
+
+    # CREATE2 Initcode
+    create2_salt = 1
+    initcode = Op.MSTORE(0, 0x112233) + Op.RETURN(0, 32) + Op.STOP()
```

Comment:
```suggestion
    initcode = Op.MSTORE(0, 0x112233) + Op.RETURN(0, 32)
```
`Op.STOP` is unnecessary here.